hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/MoveManager.java

```diff
@@ -109,8 +109,11 @@ public enum MoveResult {
   // TODO - Should pending ops notify under lock to allow MM to schedule a
   //        delete after the move, but before anything else can, eg RM?
 
-  // TODO - these need to be config defined somewhere, probably in the balancer
-  private static final long MOVE_DEADLINE = 1000 * 60 * 60; // 1 hour
+  /*
+  moveDeadline and replicationDeadline are set by ContainerBalancer.
+   */
+  private long moveDeadline = 1000 * 90 * 60;
+  private long replicationDeadline = 1000 * 60 * 60;
```

Comment:
Nit: I think it would be better to rename these to `...Timeout`, as they are time intervals.  My understanding is that deadline is the actual point in time (i.e. "now" + timeout).

Reply:
Oh yeah, thanks for pointing this out. Renaming these.